openbb_platform/core/openbb_core/app/router.py

```diff
@@ -279,14 +279,19 @@ def command(
                 },
             )
 
-            # For custom deprecation messages
+            # For custom deprecation
             if kwargs.get("deprecated", False):
-                if deprecation_message:
-                    kwargs["summary"] = deprecation_message
-                else:
-                    kwargs["summary"] = (
-                        "This functionality will be deprecated in the future releases."
-                    )
+                deprecation: OpenBBDeprecationWarning = kwargs.pop("deprecation")
+
+                class DeprecationSummary(str):
```

Comment:
I think its better to move this class in the `deprecation.py` to ensure a better code structure